app/foot_sensor/unit/foot_sensor_t.cpp

```diff
@@ -0,0 +1,38 @@
+#include "gtest/gtest.h"
+
+#include "Fibre.hpp"
+#include "DataModel.hpp"
+
+bool isConfigured;
```

Comment:
are these externed somewhere?

Reply:
oh yea in the mock hal, just ignore

Reply:
Yes they are in the related mock files I have created